### PR TITLE
CU-86a119ce2 - Neon3 Initial project bootstrapping

### PR DESCRIPTION
**Pull request by raulduartep**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-86a119ce2 Neon3 Initial project bootstrapping](https://app.clickup.com/t/86a119ce2)

**Comment by melanke** [comment]
image

File: build/icon.png
Comment (melanke) [inline build/icon.png:1]: We need to put Neon's icon

File: package.json [hunk 1]
```diff
@@ -1,317 +1,70 @@
 {
-  "name": "Neon",
-  "version": "2.21.4",
-  "main": "./main.js",
+  "name": "neon",
+  "version": "1.0.0",
```
Comment (melanke) [inline package.json:3]: 3.0.0

File: resources/icon.png
Comment (melanke) [inline resources/icon.png:1]: Neon's icon

**Review by melanke — COMMENTED** [review]
Things that can be made on the next PR. You didn't include the Layout/Menu yet, right?

**Pull request merged**